snapcraft/commands/daemon.start

```diff
@@ -189,6 +189,12 @@ if [ "${lvm_external:-"false"}" = "true" ]; then
     ln -s "${SNAP}/wrappers/run-host" "/run/bin/lvresize"
 fi
 
+# Detect presence of sideloaded lxd-agent executable.
+if [ -x "/var/snap/lxd/common/lxd-agent.debug" ]; then
```

Comment:
This should be `"${SNAP_COMMON}/lxd-agent.debug"`

Reply:
OK cool. I checked initially, and saw quite a few places that reference `/var/snap/lxd/common/` in that file so thought that was the way to do it.

Reply:
Fixed